jenkins/spark-premerge-build.sh

```diff
@@ -109,6 +109,15 @@ mvn_verify() {
     do
         TZ=$tz ./integration_tests/run_pyspark_from_build.sh -m tz_sensitive_test
     done
+
+    # test Hybrid feature
+    source "${WORKSPACE}/jenkins/hybrid_execution.sh"
+    result=$(hybrid_prepare)
+    if [ $result == 0 ]; then
```

Comment:
seems many `echo` s then from the hybrid script, the func output cannot be treated as a sign to compare. 

either echo only sign info "success" or "non-success" for the prepare func
or
try `set +e` in hybrid script and `set +e; hybrid_prepare; status=$?; set -e` in caller script 
to get correct return code

Reply:
fixed by : https://github.com/NVIDIA/spark-rapids/pull/11906/commits/9700b4078af8f496e93ef9cb26472e4dd79bae31